File: 11_November_Batch/C++/palindrome.cpp
#include<iostream>
using namespace std;
void palindrome(int n){
    int a=n,sum=0,b;
    while (n!=0)
    {
        b=n%10;
        sum=(sum*10)+b;
        n/=10;
    }
    if (a==sum)
    {
        cout<<"Palindrome";
    }
    else{
        cout<<"Not palindrome";
    }
    
}
int main(){
    int a;
    cin>>a;
    palindrome(a);
    return 0;
}

File: 4_August_Batch/Java/Palindrome.java
import java.util.*;
public class Palindrome{
    public static boolean isPalindrome(int a){
        int n=a,sum=0;
        while(a!=0){
            int b=a%10;
            sum=(sum*10)+b;
            a/=10;
        }
        if (n==sum){
            return true;
        }
        return false;
    }
    public static void main(String args[]){
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter the number: ");
        int a = sc.nextInt();
        System.out.println(isPalindrome(a));

    }
}